module-2/backend/week_one.md

```diff
@@ -4,31 +4,97 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+List the five common HTTP verbs and what the purpose is of each verb.
+
+* *get -- render/display on the html page*
+* *post -- get the parameters submitted by a form on the html page and create a new entry in the database*
+* *put -- get the parameters submitted by a form on the html page and update an existing entry in the database*
+* *delete -- html form that when submitted, prompts the database to destroy the selected entry*
+* *patch -- no idea*
+
+What is Sinatra?
+
+*Sinatra is a simple web framework to write ruby applications for the web*
+
+What is MVC?
+
+*it's a framework type that stands for Model, View, Controller*
+
+Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+*???*

```

Comment:
Generally the advantages are that developers are able to get into new projects with little prep regarding a schema. Similarly a user should see familiar routes (or at least routes that they can reasonably parse) when they see a link (e.g. I'll have a decent idea of what `http://www.restaurant-app.com/restaurants/new` is going to take me to if one of my friends sends it to me).